new-admin/jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  }
```

Comment:
Just curious: what does this do and why was it needed?

Reply:
It solved my problem (something with webpack maybe) that I could not import anything from a subfolder in index.js.
For example when testing: This won't get resolved in index.js  >   import { arraySort } from "./utils/ArraySort";
I tried, but could not get it to work without the compilerOptions. Maybe I'm missing something.